config/initializers/devise.rb

```diff
@@ -0,0 +1,15 @@
+Devise.setup do |config|
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+
+  require 'devise/orm/active_record'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.